conmon/conmon.c

```diff
@@ -450,6 +452,17 @@ int main(int argc, char *argv[])
 
 		masterfd_stderr = fds[0];
 		slavefd_stderr = fds[1];
+
+	}
+
+	/* Create a pipe to attach to the container process stdin. */
+	if (!terminal && !exec) {
+		int fds[2];
+		if (pipe(fds) < 0)
```

Comment:
Should use pipe2 + O_CLOEXEC here to avoid inheriting the pipes in the container.